src/qml_benchmarks/models/energy_based_model.py

```diff
@@ -90,11 +90,14 @@ def __init__(
         self.score_fn = score_fn
         self.mmd_kwargs = mmd_kwargs
 
-    def initialize(self, x: any = None):
-        self.fit(x[:1, ...])
```

Comment:
I removed this because I find it a bit messy; and it could take a long time in principle. A better way would be to randomly initialize the parameters of the sklearn RBM without calling fit.